board/static/board/css/basic.css

```diff
@@ -0,0 +1,100 @@
+.search_form{
+    padding-top: 40px;
+}
+
+/* Оформление панели */
+#side-checkbox {
```

Comment:
Using id in styling is not a good idea, there are classes for this. Ids are used mostly for js references.